lib/vsc/utils/generaloption.py

```diff
@@ -257,8 +257,9 @@ def take_action(self, action, dest, opt, value, values, parser):
             if action in ("add", "add_first", "add_flex",):
                 # determine type from lvalue
                 # set default first
-                values.ensure_value(dest, type(value)())
-                default = getattr(values, dest)
+                default = getattr(parser.get_default_values(), dest)
```

Comment:
add default `None` for getattr (probbaly not needed, but better safe then sorry)